index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**Writing automated tests will eliminate new bugs and speed up collaboration.**
+
+Automated tests are one of the most powerful tools in a developers toolkit. My motto forged from years of practical experience is _code that isn't tested does not work_ as expected
+
+This seminar will explain all the ways in which automated tests are essential. It will cover how to write tests, even when the code is complicated, with plenty of examples.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * What tests are.
+ * Why tests are valuable.
+ * How to write tests (including when it is tricky).
+ * When to write tests.
+ * Running tests
+
+Software & Technologies:
+
+ * [Python](https://www.python.org/){:target="_blank"} - a popular programming language
+ * [Python unittest](https://docs.python.org/3/library/unittest.html){:target="_blank"} - the built-in unit testing framework in Python
+ * [Python unittest.mock](https://docs.python.org/3/library/unittest.mock.html){:target="_blank"} - the built-in mocking framework in Python
+
+This seminar can be tweaked to use the language of your choice, or cover testing examples that resemble the kind of code you write to make this easier to apply. [Get in touch](mailto:hi@tkiley.co.uk) today to start writing automated tests today.
 
 ### "Good" Code
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Writing code is relatively easy, but writing "good" code is
+elusive. By following industry approaches over time the quality of the code will improve.
+
+**Code is a communication tool, writing good code will minimize misunderstanding between past and future developers.**
+
+My experience has taught me that good code is code that is:
+
+1. Readable
+2. Testable
+3. Maintainable
+
+This seminar will justify why writing good code is important, before providing a _large toolkit of guidelines and techniques_ that can be applied to your code to make it more readable, testable and maintainable.
+
+This seminar can be adjusted by reviewing some of your existing code to see the kind of problems that your experiencing and focusing on the techniques that will apply to your code base. [Get in touch](mailto:hi@tkiley.co.uk) today to start learning how to write better code.
```

Comment:
My inner grumpy journalist says: Passive voice.

"I can review your existing code to focus on the techniques that will help you to fix the kind of problems that you're experiencing in your code base." (or something similar)